src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureFlagOptions.cs

```diff
@@ -19,5 +19,11 @@ public class FeatureFlagOptions
         /// The time after which the cached values of the feature flags expire.  Must be greater than or equal to 1 second.
         /// </summary>
         public TimeSpan CacheExpirationInterval { get; set; } = AzureAppConfigurationOptions.DefaultFeatureFlagsCacheExpirationInterval;
+
+        /// <summary>
+        /// Selectively load only those feature flags which begin with this prefix.
+        /// The characters asterisk (*), comma (,) and backslash (\) are reserved and must be escaped using a backslash (\).
+        /// </summary>
+        public string Prefix { get; set; } = string.Empty;
```

Comment:
>Prefix [](start = 22, length = 6)

If my feature flag shows up as `.appconfig.featureflag/UI/EnableFeature` in the Azure Portal, it is not clear from the comment if the prefix expected here to get the key name as `EnableFeature`  is `.appconfig.featureflag/UI/` or `UI/`. It might help to clarify that it is the latter by including an example in the summary comments.